src/app/forecast/forecast.component.ts

```diff
@@ -46,7 +49,33 @@ export class ForecastComponent implements OnInit {
       .subscribe((forecast) => (this.forecast = forecast));
   }
 
+  isForecastHistoryEnabled () : boolean {
```

Comment:
Using functions in template is bad practice because it dependent on often rendering, so try to do that:

get isForecastHistoryEnabled(): boolean {
...
}

and in template use just isForecastHistoryEnabled like variable